echo exercise

tsjoberg opened the pull request.

tsjoberg committed:
echo exercise

tsjoberg committed:
fix jshint semicolon

afeld commented on echo.js:
```diff
@@ -1,5 +1,7 @@
 function echo(){
-  // ...
+  for (i=0;  i < arguments.length; i++){
```
You have one problem with this line... what is it?

tsjoberg replied:
is it a functional or syntax issue? The only thing I can see right now is there is an extra space between the first semicolon and i < arguments.length (just committed to remove space)?

tsjoberg replied:
Please check newest commit - removed global "i" var which I believe is the problem you have identified

tsjoberg committed:
removed extra space

tsjoberg committed:
remove global "i" var + cache arg length

tsjoberg commented:
Read up on efficient techniques on for loops and added arg.length into a variable (so it only calculates once) and added var for i to properly scope the variable (believe this was the original error you first commented on).

afeld commented:
Awesome, thanks for the fixes!
